DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/CourseDelegates/CourseDelegateViewModelFilterOptions.cs

```diff
@@ -89,19 +91,18 @@ private static string GetFilterValueForAdminField(DelegateCourseAdminField deleg
             return FilteringHelper.BuildFilterValueString(filterValueName, filterValueName, propertyValue);
         }
 
-        private static IEnumerable<FilterOptionViewModel> GetCourseDelegateAdminFieldOptions(CourseAdminField adminField)
+        private static IEnumerable<FilterOptionModel> GetCourseDelegateAdminFieldOptions(CourseAdminField adminField)
         {
             if (adminField.Options.Count > 0)
             {
                 return DelegatesViewModelFilters.GetPromptOptions(adminField);
             }
 
-            var filterValueName =
-                AdminFieldsHelper.GetAdminFieldAnswerName(adminField.PromptNumber);
+            var filterValueName = $"{AdminFieldsHelper.GetAdminFieldAnswerName(adminField.PromptNumber)}({adminField.PromptText})";

```

Comment:
Can we pull the knowledge that a filter value name looks like `thing(name)` into a method the filtering helper?
Some configurable-prompt-specific method that takes the prompt variable name and the prompt name and returns the filter name. Commonise with similar code in DelegatesViewModelFilters. 
It won't make the code more concise but it will help consolidate the knowledge about what a filter string looks like.